packages/fxa-payments-server/src/routes/Product/SubscriptionUpgrade/index.tsx

```diff
@@ -137,16 +114,17 @@ export const SubscriptionUpgrade = ({
           className="product-payment product-upgrade"
           data-testid="subscription-upgrade"
         >
-          <div
-            className="subscription-update-heading"
-            data-testid="subscription-update-heading"
-          >
-            <Localized id="product-plan-change-heading">
-              <h2>Review your change</h2>
-            </Localized>
-            <p className="subheading"></p>
-          </div>
-
+          {!isMobile ? (
```

Comment:
This is probably the only really relevant change. The rest of the changes above is just removing code that isn't used anymore.